Kitodo/src/main/java/org/kitodo/production/services/command/KitodoScriptService.java

```diff
@@ -22,11 +23,17 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
+import java.util.Optional;
+import java.util.function.BiConsumer;
+import java.util.function.Function;
 import java.util.stream.Collectors;
 
 import org.apache.commons.lang.text.StrTokenizer;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.util.Strings;
```

Comment:
I would prefer the `isEmpty` method from `StringUtils` class of the `apache-commons`.`lang3` package (or the old lang package if the old package should be used future and replacement is not planed in near future)  instead a string checking method from Log4J.